vue-components/src/components/DateInput.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-DateInput' ]"
+	>
+		<InputWithExtender
+			:label="label"
+			:value="value"
+			:feedback-type="feedbackType"
+			:placeholder="placeholder"
+			:disabled="disabled"
+			@input="$emit( 'input', $event )"
+		>
+			<template v-slot:default>
```

Comment:
This also has to reproduce the suffix slot

Reply:
Done.